samples/emoji-search/ios/app/EmojiSearchApp/NoopRuntimeLayoutModifier.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import shared
+
+class NoopRuntimeLayoutModifier: Redwood_runtimeLayoutModifier {
```

Comment:
Should find a way to not need this

Reply:
It shouldn't be needed today. You can use the companion object of `LayoutModifier` as the default instance.

Reply:
Done.